src/api_manager/method_impl.h

```diff
@@ -154,6 +154,16 @@ class MethodInfoImpl : public MethodInfo {
 
   void ProcessSystemQueryParameterNames();
 
+  bool escape_binding() const override {
```

Comment:
should this be called escape_binding()? 

maybe remain_escape_characters?

Reply:
changed it to keep_binding_escaped()